src/widget/form/settings/avform.cpp

```diff
@@ -63,7 +63,7 @@ AVForm::AVForm() :
     connect(bodyUI->videoModescomboBox, qcbxIndexChangedInt, this, &AVForm::onVideoModesIndexChanged);
 
     connect(bodyUI->filterAudio, &QCheckBox::toggled, this, &AVForm::onFilterAudioToggled);
-    connect(bodyUI->rescanButton, &QPushButton::clicked, this, [=](){getAudioInDevices(); getAudioOutDevices();});
+    connect(bodyUI->rescanButton, &QPushButton::clicked, this, [=](){getAudioInDevices(); getAudioOutDevices(); getVideoDevices();});
```

Comment:
Can you please split the commands into separate lines (-> easier to read, debug etc.).